### ISSUE TITLE
Possible to graph out number of requests over time?

### ISSUE THREAD

**Issue by vidhu** [allinurl/goaccess#1861]
I'm looking for a way to graph out the numbers of requests over time
i.e x-axis: time, y-axis: request count per second

Bonus if its possible for each route to be an individual series

Is this possible?

**Comment by allinurl** [comment]
Essentially a time-series report?

**Comment by leeoniya** [comment]
shameless plug if there's any interest in embedding an interactive multi-timeseries in the html report: https://github.com/leeoniya/uPlot

166,650 point demo: https://leeoniya.github.io/uPlot/bench/uPlot.html

**Comment by vidhu** [comment]
@allinurl yes, exactly! Currently I can see the number of request over days. I personally use this tool to view requests in real time and would love to see a time series of the routes by the second/minute
